packages/manager/apps/identity-access-management/src/pages/manageTokens/listing/ManageTokensListing.page.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import {
+  BaseLayout,
+  ManagerButton,
+  ChangelogButton,
+  HeadersProps,
+  Notifications,
+} from '@ovh-ux/manager-react-components';
+
+import { CHANGELOG_CHAPTERS, CHANGELOG_LINKS } from '@/constants';
+import { useGetIamUser } from '@/data/hooks/useGetIamUser';
+import { useParam } from '@/hooks/useParam';
+import { useIamUserTokenList } from '@/data/hooks/useGetIamUserTokens';
+import { Datagrid } from '@ovh-ux/manager-react-components';
+import { useDatagridColumn } from '@/pages/manageTokens/listing/useDatagridColumn';
+
+import { ManageTokensBreadcrumb } from '@/pages/manageTokens/components/ManageTokenBreadcrumb.component';
+
+export default function ManageTokensListing() {
+  const { t } = useTranslation('manage-tokens');
+  const { shell } = useContext(ShellContext);
+  const userId = useParam('userId');
+
+  const header: HeadersProps = {
+    title: t('iam_user_tokens_title'),
+    description: ((
+      <div>
+        <span className="mr-8">{t('iam_user_tokens_user_name')}</span>
+        <strong>{userId}</strong>
+      </div>
+    ) as unknown) as string,
+    changelogButton: (
+      <ChangelogButton links={CHANGELOG_LINKS} chapters={CHANGELOG_CHAPTERS} />
+    )
+  };
+
+  const { isLoading: isUserLoading, isError: isUserError } = useGetIamUser(userId);
+
+  const columns = useDatagridColumn();
+  const {
+    flattenData,
+    fetchNextPage,
+    hasNextPage,
+    isLoading,
+    sorting,
+    setSorting,
+    search,
+  } = useIamUserTokenList({ userId, columns, pageSize: 10 });
+
+  const handleReturnLink = () => {
+    shell?.navigation.navigateTo('', '/iam/identities/users', {});
+  };
+
+  if (isUserError) {
+    throw new Error(`Could not fetch user '${userId}'.`);
+  }
+
+  return (
+    <BaseLayout
+      header={header}
+      backLinkLabel={t('iam_user_tokens_return_to_identity_list')}
+      onClickReturn={handleReturnLink}
+      breadcrumb={<ManageTokensBreadcrumb />}
+      message={<Notifications />}
+    >
+      <React.Suspense>
+        {!isUserLoading && (
+          <Datagrid
+            topbar={
```

Comment:
Question:
Is React.Suspense usefull here?
There is no fallback defined  while the children loads and I think the point of suspense is to avoid having the checks for loadings. I don't really use React.Suspense but here it feels a little like you could just use a simple div instead for the same effect.